scripts/src/java/org/oppia/android/scripts/XMLSyntaxCheck.kt

```diff
@@ -0,0 +1,146 @@
+package org.oppia.android.scripts
+
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.parsers.DocumentBuilder
+import java.io.File
+import org.xml.sax.SAXParseException
+import org.xml.sax.ErrorHandler
+import org.oppia.android.scripts.ScriptResultConstants
+
+class XMLSyntaxCheck {
```

Comment:
add kdocs

Reply:
done

Reply:
I apparently need to reopen this since GitHub now longer allows multiple conversations for a single line of code (see: https://github.community/t/please-re-add-multiple-comments-per-line/187397).

Nit on naming: ``XmlSyntaxCheck`` per camel-casing rules: https://developer.android.com/kotlin/style-guide#camel_case

Reply:
This is corrected now.